src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageResponseCommentUiTest.java

```diff
@@ -41,14 +41,14 @@ public void testFeedbackResponseCommentActions() throws IOException {
 
         resultsPage.clickCommentModalButton("-2-1-0");
         resultsPage.addFeedbackResponseCommentInCommentModal("showResponseCommentAddForm-2-1-0", "test comment 1");
-        resultsPage.verifyCommentRowContent("-2-1-0-1", "test comment 1", "Teammates Test");
+        resultsPage.verifyFeedbackResponseCommentAdded("-2-1-0-1", "test comment 1", "Teammates Test");
```

Comment:
Sorry I missed out this:
`// TODO: all instances of this should do an immediate check instead of waiting for the comment to be added.`

So `verifyFeedbackResponseCommentAdded` should be `waitForFeedbackResponseCommentAdded`